src/inference/src/dev/threading/cpu_streams_executor.cpp

```diff
@@ -114,7 +114,12 @@ struct CPUStreamsExecutor::Impl {
             }
 #if OV_THREAD == OV_THREAD_TBB || OV_THREAD == OV_THREAD_TBB_AUTO
             if (_impl->_config.get_name().find("StreamsExecutor") == std::string::npos) {
-                set_cpu_used(_cpu_ids, NOT_USED);
+                try {
+                    set_cpu_used(_cpu_ids, NOT_USED);
+                } catch (const ov::Exception&) {
+                    // Destructor should not throw - catch needed for static analysis.
+                    // CPU::CPU() won't throw here as cpu_info() is called from Stream constructor.
+                }
```

Comment:
if we don't handle this exception, it can lead to inconsistent state of CPUStreamsExecutor.
I'm not very familiar with this code but should we stop working in case of throwing from set_cpu_used?

Reply:
Current solution (even without the above patch) is safe. However it's confusing so refactor should be considered - CVS-146130.